packages/spotlight/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The following is a curated list of changes in the Enact spotlight module, newest
 
 ## [unreleased]
 
+### Added
```

Comment:
Let's sync up your branch with the lates develop.

Reply:
done